docs/source/branch.rst

```diff
@@ -0,0 +1,52 @@
+
+The Branches File
+=================
+
+When trying to answer the research questions that drove the construction of a model and a configuration, it is often
```

Comment:
Both want to vary parameters, but also capture uncertainty.  Capturing uncertainty should be the first topic, since it's simpler (doesn't involve counterfactuals, just repeated runs of the same sim).

Outline of what should be here:

Section on uncertainty:
Subsection on varying parameter uncertainty with an example branches file with just the input draw count
Subsection on varying stochastic uncertainty with an example branches file with just the random seed count
- Note about how varying stochastic uncertainty allows aggregations, effectively letting us increase the sample size for a given input draw. (e.g. 1 sim with 1M people == 100 sims with 10k people).
Subsection on varying both at the same time with description on how we arrive at the number of runs and example model spec w/ both input_draw_number and random_seed_count  args.

Section on parameter variations (or scenarios):
Intro: "the branches section defines a set of scenarios.  We'll describe a number of ways you might want to construct simple or complex scenario specifications..."
Subsection with single parameter variation and example model spec with just the branches key.  
Subsection about how this combines with the input_draw_number/random_seed_count.
Subsection with variation of two parameters.  Explanation of how this turns into multiple scenarios.

Reply:
Final section on complex configs (use shigella example)